Bottom: 27db12b0845f1cb61171bd3247bf3a5217cf9a9d
Top:    149d3f6e5a4dfff9b203b710ecdb6488dbe94f68
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-03-02 17:14:12 +0000

Refresh of lib-fix-make-distcheck

---

diff --git a/lib/Makefile.am b/lib/Makefile.am
index d5a62f1..ae1b485 100644
--- a/lib/Makefile.am
+++ b/lib/Makefile.am
@@ -57,8 +57,11 @@ ${NEWLIB}.stamp : footprint.pcb ${NEWLIB_DEPS}
 		${PREVIEW} --pcb "${PCB}" --awk "${AWK}"
 	echo '${PREVIEW_MSG}' > $@
 
+# Remove any rebuilt footprints
+if BUILD_PCBLIB_NEWLIB
 distclean-local:
 	rm -fr ${NEWLIB}
+endif
 
 install-data-local: ${NEWLIB}.stamp
 	if test -d ${NEWLIB} ; then \
